### PR TITLE
MTDSA-4677 Updated Metadata reads

### PR DESCRIPTION
**Pull request by narendravijayarao**

### CODE REVIEW

File: app/v1/models/response/retrieveBsas/selfEmployment/Metadata.scala [hunk 1]
```diff
@@ -37,7 +37,7 @@ object Metadata {
     (JsPath \ "inputs" \ "incomeSourceType").read[IncomeSourceType].map(_.toTypeOfBusiness) and
       (JsPath \ "inputs" \ "incomeSourceId").readNullable[String] and
       JsPath.read[AccountingPeriod] and
-      (JsPath \ "metadata" \ "taxYear").read[String].map(DesTaxYear.fromDes) and
+      (JsPath \ "metadata" \ "taxYear").read[Int].map(DesTaxYear.fromDesIntToString) and
```
Comment (james-work-account) [inline app/v1/models/response/retrieveBsas/selfEmployment/Metadata.scala]: Can't we just use the old function and write something like `.read[Int].map(DesTaxYear.fromDes(_.toString))`?

File: app/v1/models/response/retrieveBsas/ukProperty/Metadata.scala [hunk 2]
```diff
@@ -35,7 +35,7 @@ object Metadata {
   implicit val reads: Reads[Metadata] = (
     (JsPath \ "inputs" \ "incomeSourceType").read[IncomeSourceType].map(_.toTypeOfBusiness) and
       JsPath.read[AccountingPeriod] and
-      (JsPath \ "metadata" \ "taxYear").read[String].map(DesTaxYear.fromDes) and
+      (JsPath \ "metadata" \ "taxYear").read[Int].map(DesTaxYear.fromDesIntToString) and
```
Comment (james-work-account) [inline app/v1/models/response/retrieveBsas/ukProperty/Metadata.scala]: as above

Comment (narendravijayarao) [reply app/v1/models/response/retrieveBsas/ukProperty/Metadata.scala]: we implemented this in individual_calculations. Just did the same...

Comment (narendravijayarao) [reply app/v1/models/response/retrieveBsas/ukProperty/Metadata.scala]: Don't merge until we update the stud responses.

**Pull request merged**